components/RadarChart.tsx

```diff
@@ -94,30 +103,48 @@ const RadarChart: React.FC<RadarChartProps> = ({ data1, data2 }) => {
     ],
   };
 
+  const maxValue = Math.max(
```

Comment:
remove this. We are only bothered with the totalCommits count. no need to make the calculations complex.